src/test/java/com/yelp/nrtsearch/server/grpc/QueryTest.java

```diff
@@ -134,6 +135,61 @@ public void testSearchV2QueryText() throws InvalidProtocolBufferException {
     LuceneServerTest.checkHits(hit);
   }
 
+  @Test
+  public void testSearchQueryResponseCompression() {
+    List<String> compressionTypes = Arrays.asList("", "identity", "gzip", "invalid");
```

Comment:
Should we test lz4 as well?

Reply:
The grpc server is build differently in testing and is only using the default registries. Maybe it is a good idea to set the custom ones there as well.